config/leviton/vrcs4.xml

```diff
@@ -0,0 +1,111 @@
+    <MetaDataItem id="0261" name="FrequencyName" type="0802">U.S. / Canada / Mexico</MetaDataItem>
+    <MetaDataItem id="0261" name="Identifier" type="0802">VRCS4-M0Z</MetaDataItem>
+    <MetaDataItem id="0243" name="ZWProductPage" type="1302">https://products.z-wavealliance.org/products/287/</MetaDataItem>
+    <MetaDataItem id="0243" name="FrequencyName" type="1302">U.S. / Canada / Mexico</MetaDataItem>
+    <MetaDataItem id="0243" name="Identifier" type="1302">VRCS4-MRZ</MetaDataItem>
+    <ChangeLog>
+      <Entry author="Jeff Brown - jeff.brown@gmail.com" date="29 June 2020" revision="1">Initial creation</Entry>
+    </ChangeLog>
+  </MetaData>
+
+  <!-- Association Groups -->
+  <CommandClass id="133">
+    <Associations num_groups="4">
+      <!-- Each button toggles an individual scene on or off.  Only one scene can be active at once.
+           If the device has a switch, then by default it will associate group 1 with the local load.
+           Do not auto associate the controller with group 1. -->
+      <Group index="1" max_associations="232" label="Button 1" auto="false"/>
+      <Group index="2" max_associations="232" label="Button 2"/>
+      <Group index="3" max_associations="232" label="Button 3"/>
+      <Group index="4" max_associations="232" label="Button 4"/>
+    </Associations>
+  </CommandClass>
+
+  <!-- Indicator Lights -->
+  <!-- Use these messages to override the indicator lights.
+
+       91 00 1D 0D 01 00 00 : Reset LEDs to locally controlled operation (default behavior).
+       91 00 1D 0D 01 FF xx : Set LEDs by OR-ing together the following bit patterns.
+
+         00000000: button 1 off
+         00000001: button 1 green
+         00010000: button 1 red
+         00010001: button 1 amber
+         00000000: button 2 off
+         00000010: button 2 green
+         00100000: button 2 red
+         00100010: button 2 amber
+         00000000: button 3 off
+         00000100: button 3 green
+         01000000: button 3 red
+         01000100: button 3 amber
+         00000000: button 4 off
+         00001000: button 4 green
+         10000000: button 4 red
+         10001000: button 4 amber
+  -->
+
+  <!-- Scene Control -->
+  <!-- Use these messages to assign scenes to the controller's buttons.
+       When no scene is assigned (default behavior), the controller sends BASIC_SET messages
+       to association groups 1 to 4 corresponding to the active button.
+       When a scene is assigned, the controller sends scene activations and level change messages
+       that offer richer control of the device's behavior.
+
+       The controller has 8 scene groups:
+         - Groups 1-4 are triggered when buttons 1-4 are pressed when their LED is not lit.
+         - Groups 5-8 are triggered when buttons 1-4 are pressed when their LED is already lit.
+
+       To configure the scenes, send SCENE_CONTROLLER_CONF_SET to associate scenes with
```

Comment:
we don't support this CC.... so your saying a user should use the SendRaw to get this working?

Reply:
Yep.  At least for now.